fgpyo/sam/__init__.py

```diff
@@ -575,6 +576,24 @@ def parse_sa_tag(tag: str) -> List["SupplementaryAlignment"]:
         """
         return [SupplementaryAlignment.parse(a) for a in tag.split(";") if len(a) > 0]
 
+    @classmethod
+    def from_read(cls, read: pysam.AlignedSegment) -> List["SupplementaryAlignment"]:
+        """
+        Construct a list of SupplementaryAlignments from the SA tag in a pysam.AlignedSegment.
+
+        Args:
+            read: An alignment. The presence of the "SA" tag is not required.
+
+        Returns:
+            A list of all SupplementaryAlignments present in the SA tag.
+            If the SA tag is not present, or it is empty, an empty list will be returned.
+        """
```

Comment:
@nh13 to follow up on our slack convo, I added some detail to the docstring to clarify the behavior when the SA tag is absent. 

lmk if you have any suggested edits to the docs before I merge